app/diagnostics.go

```diff
@@ -517,6 +521,7 @@ func (a *App) trackConfig() {
 
 	a.SendDiagnostic(TRACK_CONFIG_DISPLAY, map[string]interface{}{
 		"experimental_timezone": *cfg.DisplaySettings.ExperimentalTimezone,
+		"custom_url_schemes":    isDefault(*cfg.DisplaySettings.CustomUrlSchemes, ""),
```

Comment:
@hmhealey Not sure if this is correct

Reply:
I don't think `isDefault` will work here, but we could use `len(*cfg.DisplaySettings.CustomUrlSchemes) != 0` to check if they've changed it from the default empty array